app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
```

Comment:
think how you can optmize this line for the best performance. Hint: getting an element from dictionary by key is the fastest operation